template/compose.tpl/containers_network.yml.tpl

```diff
@@ -53,6 +53,8 @@ services:
     - tyk
   candig_server:
     image: c3genomics/candig_server
+    ports:
+    - "80:80"
```

Comment:
Why are we exposing the port outside of the compose network?

Reply:
Yea this is my bad. I will remove this and push. Good catch!

Reply:
Just removed it. I will merge all to devel and test it